app/api/users.py

```diff
@@ -43,6 +43,14 @@ class Meta:
     thumbnail = fields.Str()
     small = fields.Str()
     icon = fields.Str()
+    notification = Relationship(attribute='notification',
+                                self_view='v1.user_notification',
```

Comment:
But right now the way you have written the relationship, it rather will give user id also in the related view, rather than the notification_id